api/handlers_test.go

```diff
@@ -250,3 +252,212 @@ func TestObjectLifeline_Index_Error(t *testing.T) {
 	expected := ErrorMessage{Error: []string{"query parameter 'index' should have the '<pulse_number>:<order>' format"}}
 	require.Equal(t, expected, received)
 }
+
+func TestServer_JetDropsByPulseNumber(t *testing.T) {
```

Comment:
I would prefer to have different tests for different cases, that way you can see all failed tests after one run. What do you think?